test/fixtures/app.js

```diff
@@ -7,3 +7,15 @@ export const App = () => (
 		<Route path="/about" component={() => <div>ABOUT</div>}/>
 	</>
 );
+
+export const Login = () => console.log('lelele') || (
```

Comment:
Are you supposed to leave the console.log here and in line 19 ?

Reply:
Good catch 😅